### ISSUE TITLE
在inpaint模型中 i2v 和 v2v 推理的时候信息嵌入方式的不同 ，对结果是否会有影响？

### ISSUE THREAD

**Issue by laulampaul2** [aigc-apps/CogVideoX-Fun#74]
您好，在 pipeline_cogvideox_inpaint.py 中我发现predict v2v和predict i2v的处理方式是不同的

具体来说
v2v的时候 if (mask_video == 255).all(): 成立，
masked_video_latents = torch.zeros_like(latents)
inpaint_latents = torch.cat([mask_input, masked_video_latents_input], dim=2)

这个时候masked_video_latents_input不含有输入video的信息
信息是在 latents_outputs = self.prepare_latents（xx）嵌入到latents当中，因为这个时候is_strength_max=0.7

在i2v的时候，
图像的信息通过，masked_video = init_video * (mask_condition_tile < 0.5) + torch.ones_like(init_video) * (mask_condition_tile > 0.5) * -1输入到masked_video_latents_input，原始latents当中没有嵌入image的信息。

v2v的方式似乎跟宣称的嵌入方式不太一样，masked_video_latents_input不含有输入video的信息会对最终结果有多大影响呢？
is_strength_max（denoising_strength）的取值的含义是什么？
train.py代码中输入是图像和视频的时候分别采用的是哪种嵌入方式？

谢谢！

**Comment by bubbliiiing** [comment]
这里应该是支持两种video输入latents的方式，
一种是SD常用的inpaint方案，原输入视频加一定的噪声后作为初始latents，通过denoising_strength控制。
如果期望使用inpaint的方式进行v2v，比如使用mask，则会通过conv in注入视频信息。

**Comment by bubbliiiing** [comment]
我个人感觉两个并不冲突

**Comment by bubbliiiing** [comment]
我先关闭此issue，如果不理解可以再次打开。
